test/ggsn_SUITE.erl

```diff
@@ -2089,7 +2089,7 @@ gy_ccr_asr_overlap(Config) ->
     ResponseFun =
 	fun(Request, Result, Avps, SOpts) ->
 		Self ! {'$response', Request, Result, Avps, SOpts} end,
-    AAAReq = #aaa_request{from = ResponseFun, procedure = {gy, 'ASR'},
```

Comment:
Whatever triggered that change, that doesn't work.

The procedure is named in the config, changing that to the actual handler is IMHO not correct.

Reply:
Done